core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -92,6 +92,11 @@ export class TopicEditorTabComponent implements OnInit, OnDestroy {
   uncategorizedEditOptionsIndex: number;
   subtopicEditOptionsAreShown: number;
   skillOptionDialogueBox: boolean = true;
+  MAX_CHARS_IN_TOPIC_NAME: number;
+  MAX_CHARS_IN_TOPIC_DESCRIPTION: number;
+  maxWebTitleFrag: number;
+  minWebTitleFrag: number;
+  MAX_CHARS_IN_META_TAG_CONTENT: number;
```

Comment:
Where is this being used?

Reply:
These are used in **topic-editor-tab.directive.html**